src/util/logger.ts

```diff
@@ -154,3 +154,10 @@ export function censor(obj: any, censorList: CensorKeyValue[], throwOnError = fa
   })
   return JSON.parse(result)
 }
+
+export const censorLogs = (logFunc: () => void) => {
+  if (process.env['LOG_CENSOR'] === 'false') {
```

Comment:
We should change the name of the setting to something more specific, e.g. CENSOR_SENSITIVE_LOGS